examples/secp256k1-transfer/src/lib.ts

```diff
@@ -0,0 +1,119 @@
+const CKB_INDEXER_URL = "https://testnet.ckb.dev/indexer";
+const rpc = new RPC(CKB_RPC_URL);
+const indexer = new Indexer(CKB_INDEXER_URL, CKB_RPC_URL);
+export const MIN_CELL_CAPACITY = BI.from(61 * 1e8);
+export type Account = {
+  lockScript: Script;
+  address: Address;
+  pubKey: string;
+};
+
+export const generateAccountFromPrivateKey = (privKey: string): Account => {
+  const pubKey = hd.key.privateToPublic(privKey);
+  const args = hd.key.publicKeyToBlake160(pubKey);
+  const template = AGGRON4.SCRIPTS["SECP256K1_BLAKE160"]!;
+  const lockScript = {
+    code_hash: template.CODE_HASH,
+    hash_type: template.HASH_TYPE,
+    args: args,
+  };
+  const address = helpers.encodeToAddress(lockScript, { config: AGGRON4 });
+  return {
+    lockScript,
+    address,
+    pubKey,
+  };
+};
+
+/**
+ * calculate the transaction fee by skeleton inputs and outputs
+ * @param skeleton unsigned transaction skeleton
+ * @returns the transaction fee, it is from `sum(skeleton inputs capacity) - sum(skeleton outputs capacity)`
+ */
+export function calculateTransactionFee(skeleton: helpers.TransactionSkeletonType) {
+  const inputs = skeleton.inputs.reduce((acc, cur) => acc.add(cur.cell_output.capacity), BI.from(0));
+  const outputs = skeleton.outputs.reduce((acc, cur) => acc.add(cur.cell_output.capacity), BI.from(0));
+  return inputs.sub(outputs);
+}
+/**
+ * fetch all cells and calculate the sum of their capacities
+ */
+export async function fetchAddressBalance(address: string): Promise<BI> {
+  let balance = BI.from(0);
+
+  for await (const cell of indexer.collector({ lock: helpers.parseAddress(address, { config: AGGRON4 }) }).collect()) {
+    balance = balance.add(cell.cell_output.capacity);
+  }
+
+  return balance;
+}
+
+interface Options {
+  targets: {
+    address: string;
+    amount: BIish;
+  }[];
+  privKey: string;
+}
+
+/**
+ * create an unsigned 1:N transaction skeleton
+ */
+export async function createUnsignedTxSkeleton(options: Options) {
+  const transferFromInfo = generateAccountFromPrivateKey(options.privKey);
+
+  let txSkeleton = helpers.TransactionSkeleton({ cellProvider: indexer });
+  for (const target of options.targets) {
+    txSkeleton = await commons.common.transfer(
+      txSkeleton,
+      [transferFromInfo.address],
+      target.address,
+      target.amount,
+      transferFromInfo.address,
+      undefined,
+      { config: AGGRON4 }
+    );
+  }
+
+  txSkeleton = await payFeeByFeeRate(txSkeleton, [transferFromInfo.address], 1000, undefined, { config: AGGRON4 });
+  return txSkeleton;
+}
+
+/**
+ * sign a txSkeleton and send it to test network
+ * @returns Promise with transaction hash
+ */
+export async function transfer(txSkeleton: helpers.TransactionSkeletonType, privateKey: string): Promise<string> {
+  txSkeleton = commons.common.prepareSigningEntries(txSkeleton);
+  const message = txSkeleton.get("signingEntries").get(0)?.message;
+  const Sig = hd.key.signRecoverable(message!, privateKey);
```

Comment:
```suggestion
  const sig = hd.key.signRecoverable(message!, privateKey);
```

Reply:
Rename it to `signature`. But why previous `Sig` name appears in master branch?